components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -31,21 +32,47 @@ export namespace Project {
             creationTime: new Date().toISOString()
         };
     }
-}
 
-export interface ProjectInfo extends Project {
-    lastPrebuild?: PrebuildInfo;
+    export interface Overview {
+        branches: BranchDetails[]
+    }
+
+    export interface BranchDetails {
+        name: string;
+        branchUrl: string;
```

Comment:
**Nit**: This is the only field of `BranchDetails` that has the `branch` prefix. Maybe it's not needed?

```suggestion
        url: string;
```

Reply:
fair point.